### PR TITLE
navigation bar added in header

### PR DESCRIPTION
**Pull request by RitikPandey1**
Issue #350

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...nt":0,"usedDeprecatedRules":"26"},{"filePath":"39","messages":"40","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"41","messages":"42","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},"E:\\open source project\\doc2pen\\src\\index.js",[],"E:\\open source project\\doc2pen\\src\\App.js",[],["43","44"],"E:\\open source project\\doc2pen\\src\\reportWebVitals.js",[],"E:\\open source project\\doc2pen\\src\\containers\\editContext.js",[],"E:\\open source project\\doc2pen\\src\\components\\MainContainer\\MainContainer.js",[],"E:\\open source project\\doc2pen\\src\\components\\Header\\Header.js",[],"E:\\open source project\\doc2pen\\src\\components\\Footer\\Footer.js",[],"E:\\open source project\\doc2pen\\src\\components\\Settings\\Settings.js",[],"E:\\open source project\\doc2pen\\src\\components\\OutputComponent\\Output.js",[],"E:\\open source project\\doc2pen\\src\\components\\Dropdown\\Dropdown.js",[],{"ruleId":"45","replacedBy":"46"},{"ruleId":"47","replacedBy":"48"},"no-native-reassign",["49"],"no-negated-in-lhs",["50"],"no-global-assign","no-unsafe-negation"]
```
Comment (smaranjitghose) [inline .eslintcache:1]: Remove this

Comment (RitikPandey1) [reply .eslintcache]: @smaranjitghose  done

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Convert navbar link to black and then will merge it.

**Review by anushbhatia — COMMENTED** [review]
After this we will add the home page then you can make the routing part of it

**Comment by RitikPandey1** [comment]
> Convert navbar link to black and then will merge it.

@anushbhatia  done 👍

**Pull request merged**